distribution/changelog.txt

```diff
@@ -42,6 +42,7 @@
 - Fix: [#18442] About window background is clickable.
 - Fix: [#18449] [Plugin] Change type of listview widgets from 'scroll_view' to 'listview'.
 - Fix: [#18453] Slow walking guests don't get across level crossings in time.
+- Fix: [#18469] Land rights window issues.
```

Comment:
I know this is how the original issue was called, but I’d prefer a more descriptive entry here.

Reply:
Yeah, that's how I renamed the issue.... xD
I updated the changelog though.